rmf_human_detector/include/rmf_human_detector/HumanDetector.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef HUMANDETECTOR_HPP
+#define HUMANDETECTOR_HPP
```

Comment:
Use fully qualified name
```suggestion
#define RMF_HUMAN_DETECTOR__HUMANDETECTOR_HPP
```